src/Domain/Creators/BinaryCompactObject.hpp

```diff
@@ -473,22 +466,23 @@ class BinaryCompactObject : public DomainCreator<3> {
     using group = ExpansionMap;
   };
 
-  struct RotationAboutZAxisMap {
+  struct RotationMap {
     static constexpr Options::String help = {
         "Options for a time-dependent rotation map about the z axis"};
     using group = TimeDependentMaps;
   };
   /// \brief The initial value of the rotation angle.
   struct InitialRotationAngle {
-    using type = double;
-    static constexpr Options::String help = {"Rotation angle at initial time."};
-    using group = RotationAboutZAxisMap;
+    using type = std::array<double, 3>;
+    static constexpr Options::String help = {
+        "Rotation angle about each axis at initial time."};
+    using group = RotationMap;
   };
   /// \brief The angular velocity of the rotation.
   struct InitialAngularVelocity {
-    using type = double;
+    using type = std::array<double, 3>;
     static constexpr Options::String help = {"The angular velocity."};
```

Comment:
If I'm reading this all correctly, the input file expects something like Euler angles for the initial angle, but uses an axis-angle formulation for the initial velocity.  Could we do axis-angle for both?  (Trying to do velocities with Euler angles sounds like a mess.)

Reply:
Yes I think your suggestion of using axis-angle for both is best. Then it's trivial to get the initial quaternion from inputs. Doing that fixes several of the other issues you raised here.